src/pyaro/timeseries/Filter.py

```diff
@@ -855,4 +858,83 @@ def filter_stations(self, stations: dict[str, Station]) -> dict[str, Station]:
         if self._max_altitude is not None:
             stations = {n: s for n, s in stations.items() if (not math.isnan(s["altitude"]) and s["altitude"] <= self._max_altitude) }
         
-        return stations
\ No newline at end of file
+        return stations
+    
+@registered_filter
+class RelativeAltitudeFilter(StationFilter):
+    """
+    Filter class which filters stations based on the relative difference between
+    the station altitude, and the model topography altitude.
+
+    https://github.com/metno/pyaro/issues/39
+    """
+    def __init__(self, topo_file: str | None = None, topo_var: str = "topography", rtol: float = 1):
+        """
+        :param topo_file : A .nc file from which to read model topography data.
+        :param topo_var : Name of variable that stores altitude.
+        :param rtol : Relative toleranse.
```

Comment:
Sorry, unclear specification, this is not the relative tolerance, it is the relative height difference, `rdiff`.

add units: Relative height difference in m.